cli/src/main/java/com/devonfw/tools/ide/url/updater/AbstractUrlUpdater.java

```diff
@@ -441,6 +446,9 @@ private void doUpdateStatusJson(boolean success, int statusCode, UrlVersion urlV
 
       logger.info("For tool {} and version {} the download verification succeeded with status code {} for URL {}.", tool,
           version, code, url);
+
+      urlErrorState.updateVerifications(true);
```

Comment:
We need probably another place for this function. Currently, I have commented out `updateExistingVersions` in the `update` function. If not, the number of verification is doubled, because `doUpdateStatusJson` is called twice.